tests/functional/test_api.py

```diff
@@ -105,3 +106,29 @@ def test_api_command_query_params_multiple_become_list(run_line):
     parsed_query_string = urllib.parse.parse_qs(parsed_url.query)
     assert list(parsed_query_string.keys()) == ["filter"]
     assert set(parsed_query_string["filter"]) == {"frobulated", "demuddled", "reversed"}
+
+
+def test_api_command_with_scope_strings(monkeypatch, client_login, run_line):
+    load_response("cli.api.transfer_stub")
+    load_response("auth.oauth2_client_credentials_tokens")
+
+    run_line("globus api transfer get /foo --scope-string foobarjohn")
+
+    token_grant = [
+        kall for kall in responses.calls if kall.request.url.endswith("/token")
```

Comment:
Micro comment:
I'm knot shore what two kall my reactshun to this spelleng. But it fits with my liking for `klass` as a `class` alternative, so :shrug:

Reply:
Hmm, yeah this is definitely a practice that's in the same place in my head as `klass`. I'm not sure why though, I thought `call` (lowercased) was a namespaced word by `pytest` or`requests` or some standard testing library.

Looking for it now though I can't find the term that I'd be avoiding collisions this way...